app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
```

Comment:
- [ ] As required by this module, kindly modify the views in devise for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md).

**Your login view:**
![image](https://user-images.githubusercontent.com/12745474/192667779-04416c3d-99b3-465a-b933-3fe82dd79315.png)

**Wireframe login view:**
![image](https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png)